xml/admin_ceph_gateway.xml

```diff
@@ -3279,365 +3211,211 @@ rgw_zone=us-east-1</screen>
-     </screen>
-    </step>
-    <step>
-     <para>
-      Update the period to make the changes take effect.
-     </para>
-<screen>
-&prompt.cephuser;<command>radosgw-admin</command> period update --commit
-     </screen>
-    </step>
-    <step>
-     <para>
-      Finally, restart the Ceph Object Gateway in the secondary zone.
-     </para>
-<screen>
-&prompt.root;<command>systemctl</command> restart ceph-radosgw@rgw.`hostname -s`
-     </screen>
-    </step>
-   </procedure>
-  </sect2>
- </sect1>
+  <sect2 xml:id="ceph-rgw-maintenance">
+   <title>Maintenance</title>
+
+   <sect3 xml:id="ceph-rgw-check-sync">
+     <title>Checking the Sync Status</title>
+     <para>Information about the replication status of a zone can be
+       queried with:</para>
+<screen>&prompt.cephuser;radosgw-admin sync status
+        realm b3bc1c37-9c44-4b89-a03b-04c269bea5da (gold)
+    zonegroup f54f9b22-b4b6-4a0e-9211-fa6ac1693f49 (us)
+         zone adce11c9-b8ed-4a90-8bc5-3fc029ff0816 (us-west)
+        metadata sync syncing
+              full sync: 0/64 shards
+              incremental sync: 64/64 shards
+              metadata is behind on 1 shards
+              oldest incremental change not applied: 2017-03-22 10:20:00.0.881361s
+    data sync source: 341c2d81-4574-4d08-ab0f-5a2a7b168028 (us-east)
+                      syncing
+                      full sync: 0/128 shards
+                      incremental sync: 128/128 shards
+                      data is caught up with source
+              source: 3b5d1a3f-3f27-4e4a-8f34-6072d4bb1275 (us-3)
+                      syncing
+                      full sync: 0/128 shards
+                      incremental sync: 128/128 shards
+                      data is caught up with source</screen>
+    </sect3>
+    <sect3 xml:id="ceph-rgw-metadata-master">
+      <title>Changing the Metadata Master Zone</title>
+      <important>
+        <para>Care must be taken when changing which zone is the metadata master.
+          If a zone has not finished syncing metadata from the current master
+          zone, it is unable to serve any remaining entries when promoted
+          to master and those changes will be lost. For this reason, we recommend waiting for
+          a zone’s <literal>radosgw-admin</literal> sync status to catch up on metadata sync before
+          promoting it to master.
+          Similarly, if changes to metadata are being processed by the current master zone
+          while another zone is being promoted to master, those changes are likely to be
+          lost. To avoid this, we recommend shutting down any radosgw instances on the previous
+          master zone. After promoting another zone, its new period
+          can be fetched with <literal>radosgw-admin</literal> period pull and the gateway(s) can be restarted.</para>
+      </important>
+      <para>To promote a zone (for example, zone us-west in zonegroup us) to
```

Comment:
I suggest some form of formatting to highlight the names in this, e.g. `us-west` in zonegroup `us`.